ssm/helpers.go

```diff
@@ -53,64 +57,87 @@ func addInstanceInfo(instanceID *string, tags []ec2helpers.InstanceTags, instanc
 	}
 }
 
-// RunInvocations invokes an SSM document with given parameters on the provided slice of instances
-func RunInvocations(sp *session.Pool, sess *ssm.SSM, instances []*ssm.InstanceInformation, params *invocation.RunShellScriptParameters, dryRun bool, resultsPool *invocation.ResultSafe) (err error) {
-	var commandOutput invocation.CommandOutputSafe
-	var invError error
-
-	scoChan := make(chan *ssm.SendCommandOutput)
-	errChan := make(chan error)
-
-	/*
-		In a standard deployment, SSM allows us run commands on a maximum of
-		up to 50 instances simultaneously.
+func checkInvocationStatus(ctx ssmiface.SSMAPI, commandID *string) (done bool, err error) {
```

Comment:
ctx is usually used for actual context objects within Go. There is no reference of 'client' at all within this package and the object passed is a struct that fulfills the interface for an SSM client. I think this would be much more clear if 'ctx' was changed to 'client'.

Reply:
I guess there really isn't a lot of consistency in what the SSM client is called throughout the project. In my opinion this change still makes sense even if the others would need to be changed over time since the object being passed around is the actual SSM client